tools/walletextension/rpcapi/utils.go

```diff
@@ -34,13 +34,15 @@ const (
 	ethCallAddrPadding  = "000000000000000000000000"
 
 	notAuthorised = "not authorised"
+	serverBusy    = "server busy. please retry later"
```

Comment:
should it say retry? i.e. the transaction might have actually gone through, so you wouldn't want to send again, just go into reconcilliation

Reply:
the error response is generic for all rpc calls. The tx submission has a much higher timeout, so it shouldn't really be shown there, unless there is something really wrong